cds-snc/cppd-medical-report
# Added French translations for edit buttons

# Description

https://trello.com/c/0ZLpEUXD/406-translate-links-for-edit-conditions-edit-medications-and-edit-treatments-to-french

# Test Instructions

- Navigate through the application, adding at least 1 condition, treatment, and medication
- Check for a French translation of the edit buttons on the condition, treatment, and medication list pages
